src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -52,13 +52,23 @@
             transition: box-shadow $container-transition-duration ease $container-transition-delay;
 
             &.bcs-is-focused {
-                .bcs-ActivityCard {
-                    box-shadow: 0 0 $bdl-grid-unit * 4 $bdl-gray-30;
-                    animation: bcs-ActivityItem-callout 1s ease-in-out $container-transition-delay 0 normal none; // Disabled by default
+                :not(.bcs-ActivityThread) {
+                    .bcs-ActivityCard {
```

Comment:
are you able to remove this change if you set `animation: none` on line 70? https://github.com/box/box-ui-elements/blob/420db51d099bc0c9d467a09eb8795af05195c322/src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss#L70-L72

Reply:
if by this change you mean `:not(.bcs-ActivityThread) {` then I need to remove it eitherway because it breaks the `isFocused` state for other parts of Preview